docs/devel/storage.md

```diff
@@ -31,7 +36,54 @@ This will create
 
 After a few moments the hostpath-provisioner pod should be running, we are now ready to create PVCs against it. Any requested PVCs will be stored in `/var/run/kubevirt/hostpath/<pvc-name>-<hash>`. The PV retention policy will be `delete`, which means that when you delete the PVC, the matching PV is also removed and the directory created on the host is removed as well.
 
-## Deploy CDI controller
+# Local volume based storage
+
+Local volume is in Beta since Kubernetes 1.10 and thus no longer requires you to enable it through the [feature-gates](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/) parameter of the kubelet. To enable local volume for the development environment please follow the following steps. Note local volumes WILL work on multiple nodes due to the fact that PVCs don't get bound until a POD actually uses the PVC, at which point the local volume is bound on the node the pod is running on. The following steps are based on information found in the [Local Persistent Storage User Guide](https://github.com/kubernetes-incubator/external-storage/blob/master/local-volume/README.md)
+
+## Preliminary steps
+
+Since disks or volumes cannot be added to a node after a container is started (nodes are running in containers in the development environment). The mount source will have to be some directories that exist in the container. This can be achieved by using the --bind option when mounting. This example is creating 3 mount disks, but its possible to add or remove as many as one wants, as long as there is at least 1.
+
+Make some directory based mount points:
+
+```bash
+$ ./cluster/cli.sh ssh node01
+$ sudo su
+$ mkdir /mnt/local-storage
+$ mkdir /mnt/local-storage/dev
+$ mkdir /mnt/local-storage/dev/disk1
+$ mkdir /mnt/local-storage/dev/disk2
+$ mkdir /mnt/local-storage/dev/disk3
+$ mkdir /root/disk1
+$ mkdir /root/disk2
+$ mkdir /root/disk3
+$ mount --bind /root/disk1 /mnt/local-storage/dev/disk1
+$ mount --bind /root/disk2 /mnt/local-storage/dev/disk2
+$ mount --bind /root/disk3 /mnt/local-storage/dev/disk3
+$ chmod 777 /mnt/local-storage/dev/disk1
```

Comment:
I wonder, do we have to chmod 777 or can it be lower permissions?

Reply:
It likely can be lower permissions, but since this is designed for the local development environment it doesn't much matter. If we internalize this into kubevirtci, we will have to figure out the proper permissions (or create some actual volumes that we attach during the container start and then mount properly instead of faking the mount with the --bind)